change/office-ui-fabric-react-2019-11-25-18-02-32-user-v-satgar-ExposePropForDialog.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Expose prop for dialog component close button tooltip",
```

This should be updated too
```suggestion
  "comment": "Dialog: Use closeButtonAriaLabel for close button tooltip",
```

---

done